src/components/views/project/projectDonations/QfRoundSelector.tsx

```diff
@@ -30,48 +29,70 @@ export const QfRoundSelector: FC<IQfRoundSelectorProps> = ({
 	const navigationPrevRef = useRef(null);
 	const navigationNextRef = useRef(null);
 
+	const sortedRounds = projectData?.qfRounds
+		? projectData.qfRounds
+				.filter(round => round.isActive)
+				.concat(projectData.qfRounds.filter(round => !round.isActive))
+		: [];
+
```

Comment:
```suggestion
	const sortedRounds = projectData?.qfRounds?.sort((a, b) => b.isActive - a.isActive) || [];
```

Reply:
This actually doesn't work because the variables to use for sorting need to be strings or numbers - it can't do sorting operations on booleans - are you sure this is the best way?